### ISSUE TITLE
How to use as Component

### ISSUE THREAD

**Issue by Turutupa** [GavBaros/react-tinder-cards#1]
THIS IS SIMPLY AWESOME WORK, it looks lovely!

The only problem I am having is I want to use it as a component, but if I call it within a class component (cause I am still using classic old state) the cards just appear all over the place and way bigger, not stacked one on top of the other like it should. 

Is there a way to do this?

**Comment by GavBaros** [comment]
Hey thanks! 

I haven't configured it to be made exportable as a component yet, but might in the future. 

I would suggest you edit the CSS nested selectors to try fix the enlarged cards (will edit that entire CSS structure with styled-components soon). But if it is still an issue, it might be because of being solely dependent on Hooks rather than classes. I will also investigate this later.

**Comment by Turutupa** [comment]
I think I commented too fast. It has something to do with the .map not with classes. If you render <div><Deck /></div> instead of <Deck /> it causes what I described before (huge and unorganised cards), so it is not related to classes or css. 

I wonder how this could be exported to be used as component.  

I did a NASTY workaround for the time being which I don't like at all which is rendering <Deck /> in a separate div from the rest of the react app. The app renders in a div with id='app' and the deck in a div with id='deck'. 

If you think of a better solution I am aaaaaall ears :)

Once again, thank you very much.

**Comment by Turutupa** [comment]
I think I commented too fast. It has something to do with the .map not with classes. If you render <div><Deck /></div> instead of <Deck /> it causes what I described before (huge and unorganised cards), so it is not related to classes or css. 

I wonder how this could be exported to be used as component.  

I did a **NASTY** workaround for the time being which I don't like at all which is rendering <Deck /> in a separate div from the rest of the react app. The app renders in a div with id='app' and the deck in a div with id='deck'. 

If you think of a better solution I am aaaaaall ears :)

Once again, thank you very much.

**Comment by Turutupa** [comment]
I think I commented too fast. It has something to do with the .map not with classes. If you render 'Deck component' between 2 div's it causes what I described before (huge and unorganised cards), so it is not related to classes or css. 

I wonder how this could be exported to be used as component.  

I did a **NASTY** workaround for the time being which I don't like at all which is rendering <Deck /> in a separate div from the rest of the react app. The app renders in a div with id='app' and the deck in a div with id='deck'. 

If you think of a better solution I am aaaaaall ears :)

Once again, thank you very much.
